packages/contracts-bedrock/scripts/upgrades/stage-1.4/superchain-ops-template/README.md

```diff
@@ -0,0 +1,48 @@
+# Stage 1.4 Upgrade
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Registers a new `FaultDisputeGame` with the `DisputeGameFactory` that is backed by the [`asterisc`][asterisc] FPVM and
+the [`kona`][kona] fault proof program.
+
+The proposal was:
+
+- [ ] Posted on the governance forum.
+- [ ] Approved by Token House voting.
+- [ ] Not vetoed by the Citizens' house.
+- [ ] Executed on OP Mainnet.
+
+The governance proposal should be treated as the source of truth and used to verify the correctness of the onchain operations.
+
+Governance post of the upgrade can be found at <placeholder>.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.9.0](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.9.0-rc.3) release.
```

Comment:
This doesn't currently exist. Guessing we need to land this first then tag?

Reply:
Yep!